sign/pending_requests_test.go

```diff
@@ -219,6 +241,17 @@ func (s PendingRequestsSuite) TestWaitTimeout() {
 	s.Equal(ErrSignReqTimedOut, result.Error)
 
 	// Try approving the timeouted request, it will fail
```

Comment:
unrelated, but `timeouted` is painful to the eyes, maybe you can change it to `timed out`?